lab-shelly/readme.md

```diff
@@ -0,0 +1,22 @@
+## Description:
```

Comment:
Generally, you want to write a read me that describes how your application works.